scripts/zones/Beadeaux_[S]/mobs/Da_Dha_Hundredmask.lua

```diff
@@ -1,7 +1,7 @@
 -----------------------------------
 -- Area: Beadeaux [S] (92)
 --  NM:  Da'Dha Hundredmask
--- @spawnmob 17154195
+-- @spawnmob 17154095
```

Comment:
Can just delete that line completely. While they don't exactly change often, IDs for mobs aren't set in stone either.